index.html

```diff
@@ -0,0 +1,2199 @@
+                  </h2>
+                </a>
+                <p class="card-text">
+                  Zhang Jixian told a state-backed television channel that she 
+                  did not expect the COVID-19 outbreak to be so infectious and
+                   to have mutated into a crippling pandemic.
+                </p>
+              </div>
+            </article>
+          
+          </section>
+        </div>
+
+        <div class="col-md-7 col-lg-5">
+          <section class="top-story d-sm-none d-md-block d-lg-block">
+            <a  class="first-section-title w-100" href="#">
+              <h2 class="font-weight-bolder">TOP STORY</h2>
+            </a>
+            <article class="border-0 top-story-article">
+              <div>
+                <a href="#">
+                  <img class="rounded-0 w-100" src="https://d.newsweek.com/en/full/1581101/china-aircraft-carrier-liaoning-pacific.jpg" alt="top-story">
+                </a>
+                <a href="#">
+                  <p class="politics">POLITICS</p>
+                </a>
+              </div>
+              <div class="article-body">
+                <a href="#">
+                  <h2 class="mag_link2 article-title pt-2">
+                    Chinese Carrier Sails into Pacific, Media Mock U.S. Navy's COVID-19 Woes
+                  </h2>
+                </a>
+                <p class="top-story-paragraph">
+                  The White House coronavirus response coordinator displayed a graph
+                   that shows the mortality rates of countries around the world, 
+                   and China is at the bottom with the lowest rate.
+                </p>
+              </div>
+            </article>
+          </section>
+          <article class="more-stories-article d-flex border-top pb-5 d-sm-none d-md-block d-lg-block">
+            <div class=" justify-content-end">
+              <img class="img-fluid more-stories_img" src="https://d.newsweek.com/en/full/1581093/coronavirus-america-recoveries-40000-people.jpg" alt="more stories">
+            </div>
+            <div class="d-flex flex-column ml-3">
+              <div>
+                <a href="#">
+                  <h4 class=" more-stories-title">
+                    Restauranteur Delivers Oranges to Hospital Workers Amid Coronavirus Crisis
+                  </h4>
+                </a>
+              </div>  
+              <div>
+                <p class=" more-stories-paragraph text-body">
+                  "We all have to help each other," Matthew Fermin, 
+                  owner of East Harlem restaurant Santiago's 
+                  Beer Garden, told Newsweek.
+                </p>
+              </div>
+            </div>
+          </article>
+          <section class="heros-pandemic">
+            <a  class="first-section-title w-100" href="#">
+              <h2 class="font-weight-bolder">HEROES OF THE PANDEMIC</h2>
+            </a>
+            <div class="row heros-articles">
```

Comment:
These two sections have a different look than the original page: 
The first article should be on one line, yours is not.
The heroes of the pandemic have different font-weight and color.

![Screenshot from 2020-04-22 13-00-57](https://user-images.githubusercontent.com/51926937/80018293-6b1bc200-849b-11ea-9098-c214d8135940.png)

Original:
![Screenshot from 2020-04-22 13-01-07](https://user-images.githubusercontent.com/51926937/80018329-7a027480-849b-11ea-8556-4f05bd5caeb0.png)